tests/utils/exec/rosacli/cmd_client.go

```diff
@@ -30,7 +30,11 @@ type Client struct {
 
 	// services
 	// Keep in alphabetical order
+	AutoScaler           AutoScalerService
```

Comment:
Normally I wouldn't want unrelated changes in an MR, but to be fair the comment above this says `Keep in alphabetical order` - so im fine with this change